.github/workflows/centralized-sampling-tests.yml

```diff
@@ -0,0 +1,58 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Centralized Sampling Integration Tests
+
+on:
+  push:
+    paths:
+      - 'centralized-sampling-tests/**'
+  pull_request:
+    paths:
+      - 'centralized-sampling-tests/**'
```

Comment:
This syntax will only run on PR and Pushes involving the centralized-sampling-tests folder. That way if someone tries to change the tests in some way, it will validate that their changes don't break it